pkg/apis/internal.acorn.io/v1/memory.go

```diff
@@ -52,6 +52,7 @@ var (
 	ErrInvalidAcornMemory   = errors.New("invalid memory from Acornfile")
 	ErrInvalidSetMemory     = errors.New("invalid memory set by user")
 	ErrInvalidDefaultMemory = errors.New("invalid memory default")
+	ErrInvalidWorkload      = errors.New("invalid workload name set by user")
```

Comment:
Do we call containers as `workload` in acorn?

Reply:
Acorn containers and jobs are collectively referred to as workloads.